docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -1,13 +1,14 @@
 ---
 navigation.icon: IconDirectory
 title: server
-head.title: 'server/'
+head.title: "server/"
```

Comment:
would you be able to revert the formatting changes in the rest of the file? 🙏

Reply:
done.